cmd/osv-scanner/scan/main.go

```diff
@@ -0,0 +1,231 @@
+package scan
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"slices"
+	"strings"
+
+	"github.com/google/osv-scanner/pkg/osvscanner"
+	"github.com/google/osv-scanner/pkg/reporter"
+	"github.com/google/osv-scanner/pkg/spdx"
+	"golang.org/x/term"
+
+	"github.com/urfave/cli/v2"
+)
+
+func GetCommand(stdout, stderr io.Writer, r *reporter.Reporter) *cli.Command {
```

Comment:
nit: no need for "Get" in the name. see https://google.github.io/styleguide/go/decisions#getters